### DISCUSSION TITLE
python sdk 3.2.6 ModuleNotFoundError: No module named 'deepgram' Transcription: Pre-recorded (Jupyter OK)

### DISCUSSION THREAD

**Discussion by StreamProjectsPeter** [deepgram/community#715] [General help] [answered]
On windows   VS Code/ Conda /  Environment has python 3.11.8  and deepgram-sdk     3.2.6 installed 
Transcription: Pre-recorded example WORKS in Jupyter
from deepgram import DeepgramClient, PrerecordedOptions, FileSource" works fine code works fine  successful options = PrerecordedOptions of local and remote saved mp3/mp4  

but when running code in VS Code in python same environment I get ModuleNotFoundError: No module named 'deepgram'  ??
Please can anyone point me in the right direction assume v2/v3 issue somewhere but Jupiter works?!

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by jkroll-deepgram** [comment]
Hi @StreamProjectsPeter, can you provide a full code sample or screenshot? It's possible that Jupyter is using a different kernel than the environment that has Deepgram installed. You can choose "Select Kernel" in the top right, and make sure that your expected Python environment is selected. 

You can also try installing the Deepgram SDK directly in your Jupyter notebook, with `!pip install deepgram-sdk`.

<img width="1132" alt="Screenshot 2024-04-22 at 12 46 00 PM" src="https://github.com/deepgram/community/assets/135150417/fea1a9d3-e0df-426f-9806-0289fe395b06">

**Reply by StreamProjectsPeter** [reply]
Thanks, turns out it was a bit of an environment mess.. downgraded to miniconda and created a new env, both python via VScode and a fresh jupyter for that environment solved it.

**Comment by davidvonthenen** [comment]
The Jupyter notebooks need to be updated and there is talk about perhaps even deprecating them. I know one of the Jupyter notebooks should have a note on them saying that it's out of date, but I don't know where all of them live (before my time when joining the company).

**Reply by StreamProjectsPeter** [reply]
Thanks I don't usually use jupyter was was looking at the examples and then direct in python with vscode >> various environments.. I was just out of sync thanks to poor use of Anaconda.

**Comment by StreamProjectsPeter** [comment, marked as answer]
launching jupyter in an known active environment (with deepgram installed) via command line instead of anaconda gui, helps ensure your in the right environment...
